aidants_connect_web/tests/test_views.py

```diff
@@ -250,6 +250,12 @@ def test_wrong_code_triggers_403(self):
         response = self.client.post("/token/", fc_request)
         self.assertEqual(response.status_code, 403)
 
+    def test_wrong_nonce_triggers_403(self):
+        fc_request = dict(self.fc_request)
+        fc_request["nonce"] = "wrong_nonce"
```

Comment:
This tests test the `FC as FI` views.
The  `FC as FS` don't have any tests yet.